exercises/basiccalcobject.js

```diff
@@ -1,18 +1,37 @@
 
 class Calculator{
 	constructor(){
+		this.numbers = [];
+		this.op = null;
 		//takes in nothing
 		//make storage for the operator and the numbers
 	}
-	loadOperator(  ){
+	loadOperator(operator){
+		var possibleOperators = ['+','-','*','/']
+		if (possibleOperators.includes(operator)){
```

Comment:
also there is indexOf, making an object, putting it into a string and using IndexOf, or even a switch